sql/18-log_modified_tables.sql

```diff
@@ -0,0 +1,34 @@
+
+CREATE OR REPLACE FUNCTION ver_log_modified_tables()
+RETURNS VOID
+AS $$
+DECLARE
+    v_version_table VARCHAR;
+    v_rec RECORD;
+BEGIN
+
+    FOR v_rec IN SELECT id, schema_name, table_name
+                 FROM table_version.versioned_tables
```

Comment:
Maybe better to use the function to get the tables and account for `versioned = TRUE`

Reply:
The `ver_get_versioned_tables` function does NOT return the table identifier, which we need for our case, so we cannot use it. I've pushed a commit adding the `versioned = TRUE` filter.